bin/uglifyjs

```diff
@@ -76,6 +76,7 @@ You need to pass an argument to this option to specify the name that your module
     .describe("name-cache", "File to hold mangled names mappings")
     .describe("pure-funcs", "List of functions that can be safely removed if their return value is not used")
     .describe("dump-spidermonkey-ast", "Dump SpiderMonkey AST to stdout.")
+    .describe("debug-mangling", "Use debug mode for diagnosing property mangling errors.")
```

Comment:
Please rename to `mangle-debug` to be consistent with other property mangle options `mangle-props` and `mangle-regex`.